cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -415,14 +408,12 @@ void showView(View view, boolean isFromPM) {
     }
 
     private void runPMReady(){
-        if (this.pmReady != null)
-            uiThreadHandler.postIfEnabled(GDPRConsentLib.this.pmReady::run);
+        uiThreadHandler.postIfEnabled(GDPRConsentLib.this.pmReady::run);
```

Comment:
@Suryakantj , have you tested this without setting the `onPmReady` callback ?

Reply:
@leodoin I did test without setting those callbacks and it works as expected.